src/translator_sv.h

```diff
@@ -70,16 +71,34 @@ Problem!
 * Uppdaterat den till senaste versionen 1.8.9.1
 2015/09/12
 * Fixat lite särksirvningar och inkonsekvenser
+2020/01/08
+* Uppdaterat den till senaste språkversionen 1.8.15
+
+Bytte ut Deprecated från Föråldrad till Obsolet
+
+VHDL översättningarna är kanske inte perfekta, då jag endast använt de en gång tidigare.
+Jag lämnade use clause orörd, då jag inte hittade en lämplig översättning för den.
+
+english:
+* Uppdaterat the language translation to 1.8.15
+
+Changed Deprecated from Föråldrad to Obsolet
+
+The VHDL translations may not perfect, when I only used it once before.
+I left use clause untouched when I didn't find a suitable translation for that.
+
```

Comment:
Intentions is quite clear, thanks, but I think:
- "english"
   should be
   "English"
- "Uppdaterat " 
   should be:
  "Updated"
- "The VHDL translations may not perfect, when I only used it once before."
  should be:
  "The VHDL translations may not perfect, as I only used it once before."
- "I left use clause untouched when I didn't find a suitable translation for that."
  should be:
  "I left use clause untouched as I didn't find a suitable translation for it."

Reply:
Ok, will fix that. Thanks for the review!

Reply:
Thanks for the effort!!